t/ingredients_percent.t

```diff
@@ -368,6 +356,57 @@ my @tests = (
 
 ],
 
+        [ { lc => "en", ingredients_text => "Beans (52%), Tomatoes (33%), Water, Sugar, Cornflour, Salt, Spirit Vinegar"},
+[
+  {
+    'id' => 'en:beans',
+    'percent' => '52',
+    'percent_max' => 52,
+    'percent_min' => 52,
+    'text' => 'Beans'
+  },
+  {
+    'id' => 'en:tomato',
+    'percent' => '33',
+    'percent_max' => 33,
+    'percent_min' => 33,
+    'text' => 'Tomatoes'
+  },
+  {
+    'id' => 'en:water',
+    'percent_max' => 15,
+    'percent_min' => 3,
+    'text' => 'Water'
+  },
+  {
+    'id' => 'en:sugar',
+    'percent_max' => '7.5',
```

Comment:
100-(52+33)=15

Reply:
You need to explain the law as a comment

Reply:
Added more comments in the code and tests.